### PR TITLE
Add support for telemetry debugging via VERCEL_TELEMETRY_DEBUG

### PR DESCRIPTION
**Pull request by jeffsee55**
Specify `VERCEL_TELEMETRY_DEBUG =1 vc alias list` to view a printout of the telemetry events, rather than sending them off for collection.

This no longer writes to `output.debug` as this behavior is separate from the `--debug` global flag.

### CODE REVIEW

**Review by trek — DISMISSED** [review]

File: packages/cli/src/util/telemetry/index.ts [hunk 1]
```diff
@@ -127,9 +129,11 @@ export class TelemetryEventStore {
 
   save() {
     if (this.isDebug) {
-      this.output.debug(`${LogLabel} Flushing Events`);
+      // Intentionally not using `this.output.debug` as it will
+      // not write to stderr unless the is run with `--debug`
```
Comment (onsclom) [inline packages/cli/src/util/telemetry/index.ts]:
```suggestion
      // not write to stderr unless it is run with `--debug`
```

**Review by onsclom — DISMISSED** [review]

**Review by erikareads — COMMENTED** [review]
Why does this say files changed `2`? I don't see the change that adds support for the environment variable?

**Comment by jeffsee55** [comment]
> Why does this say files changed `2`? I don't see the change that adds support for the environment variable?

Not sure what happened there, bad merge from main I guess, fixed!

File: packages/cli/src/util/telemetry/index.ts [hunk 2]
```diff
@@ -156,7 +156,9 @@ export class TelemetryEventStore {
 
   save() {
     if (this.isDebug) {
-      this.output.debug(`${LogLabel} Flushing Events`);
+      // Intentionally not using `this.output.debug` as it will
+      // not write to stderr unless it is run with `--debug`
+      this.output.log(`${LogLabel} Flushing Events`);
       this.events.forEach(event => {
         this.output.debug(JSON.stringify(event));
```
Comment (erikareads) [inline packages/cli/src/util/telemetry/index.ts]: Shouldn't this be `this.output.log` as well?

Comment (jeffsee55) [reply packages/cli/src/util/telemetry/index.ts]: Yes! It was https://github.com/vercel/vercel/pull/12209/commits/64acfdd46b23fd98b88b4c0cd44e8d224becc753, I really don't know what happened there 🤔

Comment (erikareads) [reply packages/cli/src/util/telemetry/index.ts]: We probably also want to make the follow-up conditional an `else if` to keep in line with our plan to have debugging disable sending to server discussion

**Review by erikareads — APPROVED** [review]

**Pull request merged**